src/porepy/numerics/fv/mpfa.py

```diff
@@ -131,10 +132,22 @@ def discretize(self, g, data):
         # ambient dimension towards the end of this function (see construction of
         # vector_source_glob). This leads to a few instances of if g.dim == 1 here
         # and below.
-        if g.dim == 1:
-            active_vector_source = sps.csr_matrix((nf, nc * vector_source_dim))
+
+        # For the vector sources, some extra care is needed in the projections between
```

Comment:
@keileg, could you merge this comment with the previous one ("Vector source term. This is a bit involved..."), please? I' not entirely sure about how much may be deleted.